coralproject/talk
# [CORL-2181] MobileToolbar and Z-Key

## What does this PR do?
- Introduce MobileToolbar and Z-Key functionality
- Prevent capturing unnecessary Keypresses
- Added class `coral coral-mobileToolbar`

## How do I test this PR?
- Use `Z` or `Shift+Z` to go to next unread.
- Use `Shift+A` to mark all visible comments as read
- Reduce screensize to trigger the mobile toolbar
- Keypresses should not trigger when the comment stream is not in view or you're currently on an inputfield even on the parent page.
